docs/rule/no-invalid-aria-attributes.md

```diff
@@ -0,0 +1,22 @@
+# no-invalid-aria-attributes
+
+This rule checks for the use of invalid ARIA attributes, or any aria-\* property on an element that is not included in the [WAI-ARIA States and Properties spec](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties).
```

Comment:
Is there any legitimate use case for ARIA attributes not mentioned in that list? If so, we might want to mention them.  Or would this always be caused by typos?